CHANGELOG

```diff
@@ -1,4 +1,16 @@
 # Changelog
+## [0.2.8] - 2020-03-10
+### Bug Fixes
+ - Fix adding slave to newly create bridge causes bridge link down.
+ - Fix cannot switch bond mode from 1(active-backup) to 5(balance-tlb)
```

Comment:
```suggestion
 - Fix switchin bond mode from 1(active-backup) to 5(balance-tlb)
```

Reply:
Actually, it should be `switching mode from 1/4 to 5/6`.

Reply:
Done, thanks!